.argo/test-yamls/argo-stress-test.yaml

```diff
@@ -0,0 +1,294 @@
+---
+type: "policy"
+version: 1
+name: "Applaix Stress Every 20 minutes #1"
```

Comment:
Can you correct the name? Should not be Applaix, or Applatix. Probably Argo.

Reply:
Will do